system-int-test/test/cmr/system_int_test/ingest/granule_bulk_update/granule_bulk_update_flow_test.clj

```diff
@@ -197,4 +199,28 @@
               (let [{:keys [status errors]} (ingest/granule-bulk-update-task-status 12345)]
                 (is (= 404 status))
                 (is (= ["Granule bulk update task with task id [12345] could not be found."]
-                       errors))))))))))
+                       errors))))))))
+    (testing "Granule bulk cleanup jobs deletes old tasks"
```

Comment:
Add a granule bulk update task that will not be cleaned up to verify the cleanup actually works properly based on the cleanup minimum age.